examples/cron/SimpleCronExample.js

```diff
@@ -0,0 +1,15 @@
+const { Reshuffle, CronConnector } = require('../..')
+const app = new Reshuffle()
+
+const connector = new CronConnector(app)
+
+const logger = app.getLogger()
+
+// eslint-disable-next-line no-unused-vars
+connector.on({ expression: '*/1 * * * * *' }, (event) => {
```

Comment:
I think we should still allow passing interval. It could be one or the other as the cron expression is not intuitive.

options could be either {interval: '5'} or { expression: '*/5 * * * * *' }
@duckranger @ashevat @gabeeden what do you think?

Reply:
I think we are going to have a separate Timer connector

Reply:
I'd rather have a single connector and allow more robust options. e.g. the `CronEventOptions` could have a `frequency: MIN | HOUR | DAY | WEEK ...` and the `on` method can translate these to the right expression. 
Could also add 'startAt' and `finishAfter` if we wanted to add more power.